src/borrower/nftfeed/wrapper.sol

```diff
@@ -0,0 +1,34 @@
+// principal.sol
+// Copyright (C) 2020 Centrifuge
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.5.3;
+
+import "./nftfeed.sol";
+
+contract NFTFeedWrapper is NFTFeed {
```

Comment:
For not modifying the core contracts we can use a wrapper contract to implement the `ceiling` and ` threshold` interface